src/stabilization_guide.md

```diff
@@ -0,0 +1,181 @@
+
+# Request for stabilization
+
+    Once an unstable feature has been well-tested with no outstanding
+    concern, anyone may push for its stabilization. It involves the
+    following steps.
+
+    - Documentation PRs
+    - Write a stabilization report
+    - FCP
+    - Stabilization PR
+
+## Documentation PRs
+
+    Prepare PRs to update documentations involing this new feature.
+    You need to submit PRs for  repositories The Reference, The Book
+    and Rust by Example.
+    Maintainers of these repositories will keep these PRs open until
+    the whole stabilization process has completed. Meanwhile, we can
+    proceed to the next step.
+
+## Write a stabilization report
+
+    Find the tracking issue of the feature, and create a short
+    stabilization report. Essentially this would be a brief summary
+    of the feature plus some links to test cases showing it works
+    as expected, along with a list of edge cases that came up and
+    and were considered. This is a minimal "due diligence" that
+    we do before stabilizing.
+
+    The report should contain:
+
+        - A summary, showing examples (e.g. code snippets) what is
+          enabled by this feature.
+        - Links to test cases in our test suite regarding this feature
+          and describe the feature's behavior on encountering edge cases.
+        - Links to the documentations (the PRs we have made in the
+          previous steps).
+        - Any other relevant information(Examples of such reports can
+          be found in rust-lang/rust#44494 and rust-lang/rust#28237).
```

Comment:
I would add one other bullet:

- The resolutions of any unresolved questions if the stabilization is for an RFC.

Reply:
done